src/main/scala/io/iohk/ethereum/ledger/Ledger.scala

```diff
@@ -42,27 +40,27 @@ class Ledger(vm: VM) extends Logger {
     * This function runs transaction
     *
     * @param block
+    * @param blockchain
     * @param storages
-    * @param stateStorage
+    * @param validators
     */
   private def executeBlockTransactions(
     block: Block,
     blockchain: Blockchain,
     storages: BlockchainStorages,
-    stateStorage: NodeStorage):
+    validators: Validators):
```

Comment:
@LukasGasior1 should we create a separate task to work on that after this PR gets merged?